README.md

```diff
@@ -172,5 +173,30 @@ The actual sending of messages is managed via [`django-celery-beat`](https://dja
 When schedules are uploaded, once-off `PeriodicTask` objects are created for each row in the schedule.
 These will call `tasks.send_msgs` with the appropriate arguments for the chat.
 
+# Deployment
+
+## Demo / Test environment
+
+A demo environment has been stood up on Heroku using [Heroku's container support](https://devcenter.heroku.com/categories/deploying-with-docker).
+The environment can be found at [http://mpact-demo.herokuapp.com/](http://mpact-demo.herokuapp.com/)
+
+Configuration for the environment can be found in the `heroku.yml` file in repository root.
+The application uses Docker containers built from the `Dockerfile.heroku` file in the repository root.
+
+The `heroku.yml` file can also be used to deploy new environments. The instructions are:
+
+1. Install the `heroku` command line tool and connect it to your account.
+2. Create a new heroku application.   
+3. Run `heroku stack:set container`
```

Comment:
What does this do? I was thinking it was where one chooses the git repo, but looks like that's step 5.

Reply:
by default, heroku tries to auto-detect your programming language and use it's own toolchain. this says "don't do that i'm using docker"